### PR TITLE
Test ui-bundle.zip from antora-ui-docs that has UX improvements for i…

### PR DESCRIPTION
**Pull request by johnsmartco**
…nteractive graphics, and related TIP edits

### CODE REVIEW

**Comment by johnsmartco** [comment]
@emeliawilkinson24 and @eric-schneider , looks good in local builds.  I'll post a coppi text link in a few minutes..

**Comment by johnsmartco** [comment]
@emeliawilkinson24 and @eric-schneider , see:

https://coppi.aws.dsinternal.org/en/graphics-test/docs/migrate/introduction.html#_migration_phases

File: docs-src/zdm-core/modules/migrate/pages/introduction.adoc [hunk 1]
```diff
@@ -39,20 +39,27 @@ An important migration prerequisite is that you already have the matching schema
...
 * **Target:** This cluster is the new environment to which you want to migrate client applications and data.
 
 For additional terms, see the xref:glossary.adoc[glossary].
 
+[TIP]
+====
+To get acquainted with the migration phases, begin by clicking the **Start** button below the diagram.
+You can optionally click the illustrated components to open a larger view. 
+Click again to return to the original view and use the navigation buttons. 
+====
```
Comment (eric-schneider) [inline docs-src/zdm-core/modules/migrate/pages/introduction.adoc:55]: I think what you have works fine, but figured I'd take a stab at an alternative approach. I'm not sure it's better, but it's another option to think about in the future. There are also hypothetical SEO and site-search benefits to putting it under its own heading.

```suggestion
=== Migration interactive diagram

Click the *Start* button on the interactive diagram below to begin a walkthrough of the migration phases.
Click the components to open a larger view.
```

**Review by eric-schneider — APPROVED** [review]
Left a comment that you can take or leave. Otherwise lgtm!

**Pull request merged**